wp-irving.php

```diff
@@ -43,6 +42,24 @@
 require_once WP_IRVING_PATH . '/inc/integrations/class-pantheon.php';
 require_once WP_IRVING_PATH . '/inc/integrations/class-wpcom-legacy-redirector.php';
 require_once WP_IRVING_PATH . '/inc/integrations/class-yoast.php';
+// Integrations Manager.
+require_once WP_IRVING_PATH . '/inc/integrations/class-integrations-manager.php';
+// Instantiate the integrations manager.
+new \WP_Irving\Integrations\Integrations_Manager();
```

Comment:
Let's create a new file at `/inc/integrations/namespace.php` which includes a bootstrap function that runs all of this code, rather than doing it in the main plugin file. See the pattern set up below for `Components` and `Templates`.